fiducial_slam/src/map.cpp

```diff
@@ -409,14 +495,18 @@ int Map::updatePose(vector<Observation>& obs, const ros::Time &time,
                 continue;
             };
 
-            if (numEsts == 0) {
-                T_mapCam = p;
-            }
-            else {
-                T_mapCam.setData(averageTransforms(T_mapCam, p));
-                T_mapCam.stamp_ = p.stamp_;
-            }
-            numEsts++;
+            // compute base_link pose based on this estimate 
+
+            if (p.variance < 0.05) {
```

Comment:
Do we still need the outlier rejection I added?  If so, we should make it a param.